client.go

```diff
@@ -113,7 +113,7 @@ type Client interface {
 	LRPList(lr string) ([]*LogicalRouterPort, error)
 
 	// Add LRSR with given ip_prefix on given lr
-	LRSRAdd(lr string, ip_prefix string, nexthop string, output_port *string, policy *string, external_ids map[string]string) (*OvnCommand, error)
+	LRSRAdd(lr string, ip_prefix string, nexthop string, output_port *string, policy *string, options map[string]string, external_ids map[string]string) (*OvnCommand, error)
```

Comment:
This would break existed client code, but I don't have a better suggestion for dealing with the DB schema change. With the effort of automatic native bindings enhancement in libovsdb by @amorenoz , I hope this can be handled in a better way with code generation. @vtolstov @amorenoz @vgozer do you have any concerns here for updating API signature in go-ovn for now?

Reply:
@hzhou8 breaking API (usually a bad thing) is not a concern for me, I can't use the existing method because of its limitations. If this is a concern for others and we don't want to introduce a new method with the same functionality (e.g., LRSRaddNew() - ughh), we could hack the current method to keep the existing signature. e.g., stuff a single map[string]string with both external_ids and options K/Vs, using 'option_' prefix + key name for the latter, so we could build the corresponding column's OvsMap (external_ids key names stored as is, options map keys stripped from 'option_' prefix).  This is also ugly, probably even worse than a new method name. But it won't break the API. Someone would have to remember to clean up later :) In general, we need API versioning/auto-binding, like you said - until then, things like these will continue to haunt us.

Reply:
Thanks @vgozer for confirming. So I will merge it as is when the other comment is addressed.